### DISCUSSION TITLE
Would it be possible to add Papago language api

### DISCUSSION THREAD

**Discussion by KingDamo17** [ogkalu2/comic-translate#35] [Ideas]
I am currently trying to add Papago to the other translators but I am not sure where I need to go in the repository in order to add the request. I just need a vague direction to go in so I can work on expanding it. Thanks

**Comment by ogkalu2** [comment]
@KingDamo17 I'm not quite sure what you're asking here. Are you asking for how to request i add a feature or have you altered the code and are looking for how to share the changes you made ?

**Reply by KingDamo17** [reply]
I am currently attempting to add some code to my version to see if I can get the API for Papago to work but it is a Korean-based translator app that I think does a decent job at giving context for manhwa. I am reading the documentation but learning where to start is always a struggle for me so any tips would be appreciated. Here is the link: https://developers.naver.com/docs/papago/papago-nmt-api-reference.md#%EC%9D%B8%EA%B3%B5-%EC%8B%A0%EA%B2%BD%EB%A7%9D-%EA%B8%B0%EB%B0%98-%EA%B8%B0%EA%B3%84-%EB%B2%88%EC%97%AD
